### ISSUE TITLE
Research dataset changes 

### ISSUE THREAD

**Issue by kamerynB** [project-lux/lux-frontend#146]
**Problem Description:**
Rob is deploying a dataset that requires a few frontend changes.

**Expected Behavior/Solution:**
The frontend will change certain labels and add a new relationship to events. This branch will not be merged into main and the deployment will be from the working branch.

**Requirements:**
On the results page:
- [x] Change `Objects` to `Files` in the tab and the results header
- [x] Change `Works` to `Dataset` in the tab and the results header and swap the tab order with Objects/Files
- [x] Change `Events` to `Projects` in the tab and the results header
- [x] Same renaming as above in the Advanced Search dropdown on the home page
- [x] Add function for `/caused_by` data point in production events (same as `/classified_as`)
- [x] For Digital Objects, can we use `member_of` instead of `carries` or `shows` to populate the links between object and work?
- [x] Low Priority: Render Identifiers on people and groups, eg https://lux-front-rdata.collections.yale.edu/view/person/dd9952d3-bdf6-4a96-8590-ae492c3f7de0 has an email address identifier
- [x] Add HAL link for link between datasets and projects on Events/Projects pages

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] ~Wireframe/Mockup - Mike~
- [ ] ~Committee discussions - Sarah~
- [ ] ~Feasibility/Team discussion - Sarah~
- [x] Backend requirements - Kam add HAL link
- [x] Frontend requirements - TBD
- [ ] ~Questions~

**UAT/LUX Examples:**

HAL link examples:
- https://lux-front-rdata.collections.yale.edu/view/activity/80a17e21-bb72-46d9-844b-183946a1ebf5
- https://lux-front-rdata.collections.yale.edu/view/activity/6b27737e-90f7-4ca6-9175-ac2e0159a7f5

Results page:
- https://lux-front-rdata.collections.yale.edu/view/results/works/?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22new%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22york%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&sq=new+york

**Dependencies/Blocks:**

- Blocked By: https://github.com/project-lux/lux-middletier/issues/44

**Related Github Issues:**
N/A

**Related links:**
N/A

**Wireframe/Mockup:**
N/A

**Comment by azaroth42** [comment]
Shall we just use this issue for other minor updates? Or make a new one?

In particular, and no rush for any of this:

-[ ] Change the descriptions of Datasets, Files and Projects to read: "Records about datasets that match your search" "Records about data files such as CSVs that match your search" and "Records about projects and grants related to datasets that match your search"
-[ ] Change the Digital Object search for works header from "This file includes the following datasets" to "This file is part of the following datasets"

**Comment by roamye** [comment]
@azaroth42 @kamerynB this has an awaiting review label but its status is 'in progress'. OK to move status to reviewable or is this issue still being worked on?

**Comment by kamerynB** [comment]
@azaroth42 We can make changes in this issue. The descriptions in the first bullet come from the CMS. It looks like the rdata environment points to our existing CMS instance. I could hardcode those specific ones though. And the second bullet is only for Digital Objects and not Physical ones?

**Comment by kamerynB** [comment]
@roamye This is for Rob's research dataset. There is no need for it to be reviewed but I wanted Rob to know he could take a look at it.

**Comment by azaroth42** [comment]
Ahh huh. For a new ticket ... What happens if the CMS is down, or doesn't exist (like in another organization's adoption of LUX)?

But for this ticket -- can you just hard code them? And yes, the second is only for digital not physical -- I don't /think/ there are any physical objects in the dataset

**Comment by kamerynB** [comment]
@azaroth42 We removed the CMS dependency on the frontend before the migration.

But sounds good!

**Comment by azaroth42** [comment]
Right ... but if it gets it from the CMS, what happens when it's not there? There's a default? Or just no explanation?

**Comment by kamerynB** [comment]
There is no explanation, which is in line with what we have been doing with any config the frontend may be dependent on. The app still works, but some things may be missing.

**Comment by prowns** [comment]
Per UAT - ok to close
